cabal.project

```diff
@@ -252,8 +252,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 4fac197b6f0d2ff60dc3486c593b68dc00969fbf
-  --sha256: 1b43vbdsr9m3ry1kgag2p2ixpv54gw7a4vvmndxl6knqg8qbsb8b
+  tag: 4ae7d33231611111780e0ab36b6d648a6527f995
```

Comment:
I would like it be set past input-output-hk/ouroboros-network@173d9395c.  This will include the fix for #3568.

Reply:
Now bumped to 48e498cfea694b89f99fd92ae7a86b9fcd5f29f0